src/sql/workbench/contrib/notebook/browser/notebookActions.ts

```diff
@@ -308,11 +308,31 @@ export class RunParametersAction extends TooltipFromLabelAction {
 		const editor = this._notebookService.findNotebookEditor(context);
 		// Set defaultParameters to the parameter values in parameter cell
 		let defaultParameters = new Map<string, string>();
+		let emptyParameterCell: boolean;
 		editor.cells.forEach(cell => {
 			if (cell.isParameter) {
```

Comment:
Might want to consider breaking out after finding a parameter cell - and possibly display a warning (since having more than one parameter cell I would assume is not supported)

Reply:
We have already ensured that users cant have more than two parameter cells in a notebook when they try to set another cell as a parameter cell we inform the user. I can add a return statement once we find one though so that we don't have to iterate through each cell.

Reply:
A user can easily have a notebook they got elsewhere that doesn't adhere to this and we wouldn't detect that. You can address that in a separate PR if you wish but that's something I think we should address in general (possibly when opening a notebook as well)

Reply:
Yeah, I can look into that further if a user tagged it from another notebook viewer. I think it's a general principle from Papermill as well to follow the parameters cell format. Any additional parameters would be tagged as "Injected-Parameters".